workflows/rnaseq/main.nf

```diff
@@ -167,25 +167,6 @@ workflow RNASEQ {
             meta, num_reads ->
                 return [ meta.id, num_reads > params.min_trimmed_reads.toFloat() ]
         }
-    //
-    // Get list of samples that failed trimming threshold for MultiQC report
-    //
-    ch_trim_read_count
-        .map {
-            meta, num_reads ->
-                if (num_reads <= params.min_trimmed_reads.toFloat()) {
-                    return [ "$meta.id\t$num_reads" ]
-                }
-        }
-        .collect()
-        .map {
-            tsv_data ->
-                def header = ["Sample", "Reads after trimming"]
-                sample_status_header_multiqc.text + multiqcTsvFromList(tsv_data, header)
-        }
-        .set { ch_fail_trimming_multiqc }
-
-    ch_multiqc_files = ch_multiqc_files.mix(ch_fail_trimming_multiqc.collectFile(name: 'fail_trimmed_samples_mqc.tsv'))
```

Comment:
Is it right that this completely removed without replacement?

Reply:
Yep! It's duplicated code, a relic of some factoring-out, part of https://github.com/nf-core/rnaseq/pull/1382

Reply:
The replacement exists in the subworkflow, I think:

https://github.com/nf-core/rnaseq/blob/53023cd7db30da99c6d4a04a30f0d719bb853222/subworkflows/nf-core/fastq_qc_trim_filter_setstrandedness/main.nf#L192-L214

The deletion here removes the duplicated logic.